README.adoc

```diff
@@ -312,7 +383,7 @@ Run the Maven `package` goal to compile the test classes. Run the Maven `failsaf
 [role='command']
 ```
 mvn package
-mvn failsafe:integration-test -Dsystem.ip=[system-ip-address] -Dinventory.http.port=9081 -Dsystem.http.port=9080
+mvn failsafe:integration-test -Dsystem.ip=[system-ip-address] -Dinventory.http.port=9091 -Dsystem.http.port=9080
```

Comment:
I get the following error when running this command on Windows

```
[ERROR] Unknown lifecycle phase ".ip=172.17.0.2". You must specify a valid lifecycle phase or a goal in the format <plugin-prefix>:<goal> or <plugin-group-id>:<plugin-artifact-id>[:<plugin-version>]:<goal>. 
```

Reply:
I discussed with Gilbert and he said to leave the command as is. The guides don't claim to support PowerShell so we don't need to work around it.